packages/client/bin/cli.ts

```diff
@@ -199,12 +197,16 @@ async function runNode(config: Config) {
 }
 
 function runRpcServer(client: EthereumClient, config: Config) {
-  const { rpcport, rpcaddr } = config
+  const { rpcHttpPort, rpcWsPort, rpcaddr } = config
   const manager = new RPCManager(client, config)
   const server = new RPCServer(manager.getMethods())
-  config.logger.info(`RPC HTTP endpoint opened: http://${rpcaddr}:${rpcport}`)
-  server.http().listen(rpcport)
-
+  typeof rpcHttpPort === 'number' &&
+    config.logger.info(`RPC HTTP endpoint opened: http://${rpcaddr}:${rpcHttpPort}`)
+  typeof rpcHttpPort === 'number' && server.http().listen(rpcHttpPort)
+  rpcWsPort !== false && typeof rpcWsPort === 'number' && server.websocket({ port: rpcWsPort })
+  rpcWsPort !== false &&
+    typeof rpcWsPort === 'number' &&
+    config.logger.info(`RPC WS endpoint opened: ws://${rpcaddr}:${rpcWsPort}`)
```

Comment:
would prefer if we could rewrite this to use if() statements rather than inlined short-circuiting behavior of the `&&` operator (kind of hard to read/parse)

```suggestion
  if (typeof rpcHttpPort === 'number') {
    server.http().listen(rpcHttpPort)
    config.logger.info(`RPC HTTP endpoint opened: http://${rpcaddr}:${rpcHttpPort}`)
  }
  if (typeof rpcWsPort === 'number') {
    server.websocket({ port: rpcWsPort })
    config.logger.info(`RPC WS endpoint opened: ws://${rpcaddr}:${rpcWsPort}`)
  }
```